code-challenges.test.js

```diff
@@ -52,5 +119,84 @@ const hand2 = [5, 5, 3, 3, 4]
 const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe ("fullHouse", () => {
+    it("should return true if the array is a full house", () => {
+        expect(fullHouse(hand1)).toBe(true);
+        expect(fullHouse(hand2)).toBe(false);
+        expect(fullHouse(hand3)).toBe(false);
+    })
+})
+
 
 // b) Create the function that makes the test pass.
+
+// Psuedocode:
+// 1. Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”.
+// 2. Create a function that takes in an array of numbers and returns true if the array is a full house.
+// 3. Create a function that takes in an array of numbers and returns false if the array is not a full house.
+
+// Psuedocode:
+// 1. Create a function that takes in an array of numbers and returns true if the array is a full house.
+// for loop iteration to check if the array is a full house
+// for the length of the array of numbers
+// if the array contains the same number twice
+// else if the array contains the same number three times
+// return true
+// else
+// return false
+
+
+const fullHouse = (array) => {
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] === array[i - 1]) {
+            pair++;
+        } else if (array[i] === array[i - 2]) {
+            threeOfAKind++;
+        }
+    }
+    if (pair === 1 && threeOfAKind === 1) {
+        return true;
+    } else {
+        return false;
+    }
+}
```

Comment:
You are definitely heading in the right direction with this.  Does your test pass with this?  It looks like currently pair and threeOfAKind are not declared anywhere.  Also, consider that you can have the same numbers at different points in the array, so you may need to sort the array so the same numbers are grouped together.